crates/testing/src/task_helpers.rs

```diff
@@ -84,22 +70,22 @@ pub async fn build_system_handle(
     let memberships = Memberships {
         quorum_membership: <TestTypes as NodeType>::Membership::create_election(
             known_nodes_with_stake.clone(),
-            quorum_election_config.clone(),
+            known_nodes_with_stake.clone(),
             config.fixed_leader_for_gpuvid,
         ),
         da_membership: <TestTypes as NodeType>::Membership::create_election(
             known_nodes_with_stake.clone(),
-            committee_election_config,
+            config.known_da_nodes.clone().into_iter().collect(),
             config.fixed_leader_for_gpuvid,
         ),
         vid_membership: <TestTypes as NodeType>::Membership::create_election(
             known_nodes_with_stake.clone(),
-            quorum_election_config.clone(),
+            known_nodes_with_stake.clone(),
             config.fixed_leader_for_gpuvid,
         ),
         view_sync_membership: <TestTypes as NodeType>::Membership::create_election(
             known_nodes_with_stake.clone(),
-            quorum_election_config,
+            known_nodes_with_stake.clone(),
```

Comment:
same remark as above about moving the last call. Looks like there's a couple more spots below this. I won't mark all of them to avoid spamming.